templates/dashboard.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>TA Scheduling Tool</title>
+
+  <style>
+    * { box-sizing: border-box; }
+    body {
+      font-family: Arial, Helvetica, sans-serif;
+      background: white;
+    }
+
+    header {
+      display: flex;
+      align-items: flex-end;
+      background: white;
+        padding: 10px 20px;
```

Comment:
```suggestion
        margin: 10px 20px;
```
if this shrinks the image instead of adding room around it, ignore this suggestion